### PR TITLE
Adding datababase and data has been resolved

### PR DESCRIPTION
**Pull request by AntonBakum**

### CODE REVIEW

File: pet-api/pet-api/Domain/Entities/Product.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿namespace pet_api.Domain.Entities
+{
+   public class Product
+   {
+        public int ProductId { get; set; }
```
Comment (YevheniiDomnich) [inline pet-api/pet-api/Domain/Entities/Product.cs]: Rename it please to 'Id', it is related to the Product class and it is clear that it's a Product identifier.

File: pet-api/pet-api/Domain/Entities/Product.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public double Price { get; set; }
```
Comment (YevheniiDomnich) [inline pet-api/pet-api/Domain/Entities/Product.cs]: decimal, all types related to the money should be decimal

File: pet-api/pet-api/Domain/Entities/ProductFeedback.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿namespace pet_api.Domain.Entities
+{
+    public class ProductFeedback
+    {
+        public int ProductFeedbackId { get; set; }
```
Comment (YevheniiDomnich) [inline pet-api/pet-api/Domain/Entities/ProductFeedback.cs]: Id

File: pet-api/pet-api/Infrastructure/DAL/DatabaseContext.cs [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+{
+    public class DatabaseContext: IDatabaseContext
+    {
+        private readonly IConfiguration _configuration;
+
+        public DatabaseContext (IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+        public IDbConnection CreateConnection()
```
Comment (YevheniiDomnich) [inline pet-api/pet-api/Infrastructure/DAL/DatabaseContext.cs]: Ideally, it should be configured in the DI container in the startup project file. Your connections are not flexible, the client of the DAL dll should create the SqlConnection and MasterConnection keys in the configuration file. What if the projects already contain such keys?

File: pet-api/pet-api/Program.cs [hunk 5]
```diff
@@ -1,14 +1,29 @@
...
+using pet_api.Infrastructure.DAL.Extensions;
+using FluentMigrator.Runner;
+using System.Reflection;
 
 var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddLogging(c => c.AddFluentMigratorConsole())
+.AddFluentMigratorCore()
+        .ConfigureRunner(c => c.AddSqlServer()
+            .WithGlobalConnectionString("server=.; database=PetApiDatabase; Integrated Security=true")
```
Comment (YevheniiDomnich) [inline pet-api/pet-api/Program.cs]: it should be in application.json

File: pet-api/pet-api/LogFiles/file-2022-11-26.txt [hunk 6]
```diff
@@ -0,0 +1,14 @@
+Now listening on: https://localhost:7071
```
Comment (YevheniiDomnich) [inline pet-api/pet-api/LogFiles/file-2022-11-26.txt:1]: add logs to the gitignore

Comment (ValentynCGL) [inline pet-api/pet-api/LogFiles/file-2022-11-26.txt:1]: Du not publish logs

File: pet-api/pet-api/Infrastructure/Logging/FileLoggerExtensions.cs [hunk 7]
```diff
@@ -2,10 +2,10 @@
 
 namespace pet_api.Infrastructure.Logging
 {
-    public static class FileLoggerExtensions
+    public  static class FileLoggerExtensions
```
Comment (ValentynCGL) [inline pet-api/pet-api/Infrastructure/Logging/FileLoggerExtensions.cs]: remove extra space

File: pet-api/pet-api/Domain/Entities/Product.cs (hunk 1 above)
Comment (AntonBakum) [reply pet-api/pet-api/Domain/Entities/Product.cs]: this property has been renamed

File: pet-api/pet-api/Domain/Entities/Product.cs (hunk 2 above)
Comment (AntonBakum) [reply pet-api/pet-api/Domain/Entities/Product.cs]: property type has been changed to Decimal

File: pet-api/pet-api/Domain/Entities/ProductFeedback.cs (hunk 3 above)
Comment (AntonBakum) [reply pet-api/pet-api/Domain/Entities/ProductFeedback.cs]: this property has been renamed

File: pet-api/pet-api/Infrastructure/DAL/DatabaseContext.cs (hunk 4 above)
Comment (AntonBakum) [reply pet-api/pet-api/Infrastructure/DAL/DatabaseContext.cs]: The new format of the databaseContext has been added

File: pet-api/pet-api/Program.cs (hunk 5 above)
Comment (AntonBakum) [reply pet-api/pet-api/Program.cs]: resolved

File: pet-api/pet-api/Infrastructure/Logging/FileLoggerExtensions.cs (hunk 7 above)
Comment (AntonBakum) [reply pet-api/pet-api/Infrastructure/Logging/FileLoggerExtensions.cs]: The extra space has been removed

File: pet-api/pet-api/LogFiles/file-2022-11-26.txt (hunk 6 above)
Comment (AntonBakum) [reply pet-api/pet-api/LogFiles/file-2022-11-26.txt:1]: There are some problems with it, but I`ll try to find a possibility to resolve

Comment (AntonBakum) [reply pet-api/pet-api/LogFiles/file-2022-11-26.txt:1]: There are some problems with it, but I`ll try to find a possibility to resolve

File: pet-api/pet-api/Domain/Interfaces/IDatabaseContext.cs [hunk 8]
```diff
@@ -6,7 +6,7 @@ namespace pet_api.Domain.Interfaces
 {
     public interface IDatabaseContext
     {
-        public IDbConnection CreateConnection();
-        public IDbConnection CreateMasterConnection();
+        SqlConnection SqlConnection { get; set; }
```
Comment (YevheniiDomnich) [inline pet-api/pet-api/Domain/Interfaces/IDatabaseContext.cs:9]: You can use init; instead of set; to add some sort of encapsulation.

**Pull request merged**